backend/lib/utils.js

```diff
@@ -15,4 +15,15 @@ function computeHash(filePath, callback) {
   });
 }
 
+function getByPages(array, limit, page) {
+  let newArray = [];
+  if (array.length <= limit * (page - 1)) {
+    if (array.length <= limit * (page - 2)) newArray = [];
+  } else {
+    newArray = array.slice(limit * (page - 1), limit * page);
```

Comment:
Here logic can simplify to this:

```javascript
if (arrar.length >= limit * page) {
  newArray = array.slice(limit * (page - 1), limit * page);
}
```
What do you think?

Reply:
Cool!